rules-tests/CodeQuality/Rector/FuncCall/InlineIsAInstanceOfRector/Fixture/variable_class.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\FuncCall\InlineIsAInstanceOfRector\Fixture;
+
+class VariableClass
+{
+    /**
+     * @param class-string<\PHPStan\Type\Type> $type
+     */
+    public function run(object $object, string $type)
+    {
+        return is_a($object, $type);
```

Comment:
I think you need another test which verifies that is_a() with a non-class-string 2nd arg is skipped

Reply:
That's not needed, all variables seems fine

Reply:
If there is a bug on variable usage for 2nd arg (class arg) after transformation, then the bug is on the original is_a